README.md

```diff
@@ -50,6 +46,19 @@ Third-party Drivers:
 * [oranif](https://github.com/KonnexionsGmbH/oranif) Driver for Erlang.
 * [nimodpi](https://github.com/mikra01/nimodpi) Driver for Nim.
 
+* ## Security
+
+Please consult the [security guide](./SECURITY.md) for our responsible security
+vulnerability disclosure process
+
+## Contributing
+
+This project is not accepting external contributions at this time. For bugs or
```

Comment:
These comments need to be added to CONTRIBUTING.md.

I would just update CONTRIBUTING.md and leave the original text that was here in README: no need to repeat it.

Reply:
This comment is provided here in case you don't have a contributing file in the repository, in which case the GCAS tool skips scanning the contributing file so that the repository can pass the check. Since you want to keep this file, I will remove that comment and leave it as it was before the changes